src/compiler.adoc

```diff
@@ -1113,9 +1113,8 @@ To test it, we should write a test case using the previously mentioned `async` m
 The `done` function exposed by the `async` macro should be called after the asynchronous
 operation is finished and all assertions have run.
 
-It is very important to execute that function only once. If you execute that function twice or
-never, may cause some kind of strange behavior like executing some tests more than one time
-or skipping execution of some tests.
+It is very important to execute the `done` function only once. Omitting or
+executing it twice may cause strange behavior and should be avoided.
```

Comment:
- clarify: that function -> the `done` function
- idiom: execute twice or never -> omitting or executing twice
- simplify: discourage use of `done` without explaining undefined nature that could change (just an opinion)